x/upgrade/cli.go

```diff
@@ -0,0 +1,32 @@
+package upgrade
+
+import (
+	"github.com/spf13/cobra"
+
+	cli "github.com/cosmos/cosmos-sdk/x/upgrade/client/cli"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   upgradetypes.ModuleName,
+		Short: "Upgrade transaction subcommands",
+	}
+
+	return cmd
+}
```

Comment:
Do we need this?

Reply:
only if we decide that we do need the app module, since the [`AppModuleBasic` interface returns a pointer](https://github.com/celestiaorg/cosmos-sdk/blob/f19e20459b8bcf195167ad256de5a6ca6edc41f5/types/module/module.go#L58)